src/System.Threading.Overlapped/src/project.json

```diff
@@ -1,26 +1,13 @@
 {
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.7": {
       "dependencies": {
-        "Microsoft.NETCore.Platforms": "1.2.0-beta-24701-01",
         "Microsoft.TargetingPack.Private.CoreCLR": "1.2.0-beta-24628-03"
       }
     },
-    "netcore50": {
+    "net463": {
       "dependencies": {
-        "Microsoft.NETCore.Platforms": "1.2.0-beta-24701-01",
-        "System.Diagnostics.Debug": "4.0.11",
-        "System.Resources.ResourceManager": "4.0.1",
-        "System.Runtime": "4.1.0",
-        "System.Runtime.Extensions": "4.1.0",
-        "System.Runtime.Handles": "4.0.1",
-        "System.Runtime.InteropServices": "4.1.0",
-        "System.Threading": "4.0.11"
-      }
-    },
-    "net46": {
-      "dependencies": {
-        "Microsoft.TargetingPack.NETFramework.v4.6": "1.0.1"
+        "Microsoft.TargetingPack.NETFramework.v4.6.2": "1.0.1"
```

Comment:
yes, there is no TargetingPack for 4.6.3, so we have been using the 462 thus far.